core/camera/build.gradle.kts

```diff
@@ -94,10 +94,16 @@ dependencies {
     testImplementation(libs.mockito.core)
     androidTestImplementation(libs.androidx.junit)
     androidTestImplementation(libs.androidx.espresso.core)
+    androidTestImplementation(libs.kotlinx.coroutines.test)
+    androidTestImplementation(libs.rules)
+    androidTestImplementation(libs.truth)
 
     // Futures
     implementation(libs.futures.ktx)
 
+    // LiveData
+    implementation(libs.androidx.lifecycle.livedata)
+
```

Comment:
I can't found anywhere useing LiveData, is there I am missing point?

Reply:
It is used by cameraInfo.torchState.asFlow() in CameraXCameraUseCase.kt.

Reply:
Oh, I didn't know that `cameraInfo` had been using liveData. Thank you.